core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -164,6 +165,9 @@ public class FileSystemContext implements Closeable {
   @GuardedBy("this")
   private final RefreshPolicy mWorkerRefreshPolicy;
 
+  private volatile BlockLocationPolicy mReadBlockLocationPolicy;
+  private volatile BlockLocationPolicy mWriteBlockLocationPolicy;
```

Comment:
Discussed with @maobaolong they are put in `FileSystemContext` instead of `ClientContext` because the policies are in `alluxio-core-client-fs` module but `ClientContext` is in `alluxio-core-common` module. If they are added to `ClientContext` that will introduce a circular dependency? I think this is fine. WDYT@dbw9580 ?

Reply:
Thanks for the explanation for me.